Cabal/Distribution/PackageDescription/Parse.hs

```diff
@@ -493,6 +493,9 @@ binfoFieldDescrs =
  , listFieldWithSep vcat "extra-ghci-libraries"
            showToken          parseTokenQ
            extraGHCiLibs      (\xs    binfo -> binfo{extraGHCiLibs=xs})
+ , listFieldWithSep vcat "extra-bundled-libraries"
```

Comment:
Needs docs.

Reply:
Will see to this and the haddocks tomorrow. Thanks for the review!